chart.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var ctx = document.getElementById('myChart').getContext('2d');
+
+var parseData = JSON.parse(localStorage.allProducts);
+console.log(parseData);
+var nameTitle = [];
+var clickTitle = [];
+var viewTitle = [];
+
+function sendData () {
+  for (var i = 0; i < parseData.length; i++) {
+    nameTitle.push(parseData[i].imageName);
+    clickTitle.push(parseData[i].timesClicked);
+    viewTitle.push(parseData[i].timesDisplayed);
+  }
+}
+sendData();
+
+var myChart = new Chart(ctx, {
+    type: 'bar',
+    data: {
+        labels: nameTitle,
```

Comment:
the indentation is inconsistent on your configuration object, you are using two spaces in some places and four in others.